apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.service.js

```diff
@@ -85,18 +85,16 @@ function buildSingleExaminationTimetableItemPayload(examinationTimetableItem) {
  * @returns {Promise<void>}
  */
 export async function publish(id) {
+	const examTimetableItemsPayload = await buildExamTimetableItemsPayload(+id);
+
+	await eventClient.sendEvents(NSIP_EXAM_TIMETABLE, examTimetableItemsPayload, EventType.Publish);
+
 	const now = new Date();
-	const updatedExaminationTimetable = await examinationTimetableRepository.updateByCaseId(+id, {
+	await examinationTimetableRepository.updateByCaseId(+id, {
```

Comment:
```suggestion
	await examinationTimetableRepository.updateByCaseId(Number(id), {
```

Total nitpick. It just makes the code more readable at a glance :)

Reply:
Some other examples further down if you do decide to change this.